jupyterlab_vizier/src/service/vizier_service.ts

```diff
@@ -54,18 +54,12 @@ export function prettifyTrial(rawTrialName: string): string {
  * Class to interact with Vizier
  */
 export class VizierService {
+  // Only available in this region regardless of instance zone
+  private readonly REGION = 'us-central1';
```

Comment:
Suggest assigning this value from the shared constant or using it directly if you don't expect it to ever be an instance level value.